README.md

```diff
@@ -47,6 +47,10 @@ Note that all flags can be replaced with environment variables; for instance,
 `--dry-run` could be replaced with `EXTERNAL_DNS_DRY_RUN=1`, or
 `--registry txt` could be replaced with `EXTERNAL_DNS_REGISTRY=txt`.
 
+## There are two ways of deploying ExternalDNS:
+* Deploying to a Cluster
+* Running Locally
```

Comment:
"Running Locally" does not sound like a way of deploying to me.

Reply:
You you are right - will change it.